2-WebApp-graph-user/2-1-Call-MSGraph/Controllers/HomeController.cs

```diff
@@ -67,5 +103,15 @@ public IActionResult Error()
         {
             return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
         }
+
+        private string GetSecretFromKeyVault()

```

Comment:
Why do we have this method here? it's private and not used at all in this controller?